resharper/resharper-unity/src/Yaml/Psi/Resolve/EventHandlerSymbolFilter.cs

```diff
@@ -0,0 +1,58 @@
+using JetBrains.ReSharper.Psi;
+using JetBrains.ReSharper.Psi.Modules;
+using JetBrains.ReSharper.Psi.Resolve;
+
+namespace JetBrains.ReSharper.Plugins.Unity.Yaml.Psi.Resolve
+{
+    public class EventHandlerSymbolFilter : SimpleSymbolFilter
+    {
+        private IType myType;
+        public EventHandlerSymbolFilter(int mode, string type, IPsiModule psiModule)
+        {
+            if (mode == 2)
```

Comment:
Can we use an enum or constants instead of raw values? I don't know what 0 or 1 means here.